rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -605,7 +643,13 @@ def refresh(self, instance):
         self._load_relationships(instance)
         return instance
 
+    @with_tracing()
+    def _enable_tracing(self):
+        """Enable tracing for all SQLAlchemy engines.
+        """
+        SQLAlchemyTracingHelper.enable_tracing_for_all_engines()
```

Comment:
why there is a need for this and for the decorator

Reply:
This enables SQL Engines tracing. Otherwise you won't see any queries.